generate/canoto.go

```diff
@@ -136,7 +137,7 @@ ${sizeCache}${oneOfCache}}
 // CanotoSpec returns the specification of this canoto message.
 func (*${structName}${generics}) CanotoSpec(types ...reflect.Type) *${selector}Spec {
 	types = append(types, reflect.TypeOf(${structName}${generics}{}))
-	var zero ${structName}${generics}
+${zero}
 	s := &${selector}Spec{
```

Comment:
Super nit picky, but it feels a bit odd to me that with `zero` there isn't a newline and without `zero` there is a newline... It feels like we should probably either always or never have the newline.
```suggestion
${zero}	s := &${selector}Spec{
```
(And append a newline to the value of `zero` if it is set)

Reply:
went with no newline